src/test/resources/pages/instructorCoursesAddInvalidIdFailed.html

```diff
@@ -112,7 +112,7 @@ <h2>
             Software Engineering $^&*()
           </td>
           <td data-date-stamp="${datetime.now.iso8601utc}" data-original-title="${datetime.now}" data-toggle="tooltip" id="coursecreateddate2">
-            ${datetime.now.courses}
+            14 May 2018
```

Comment:
Same here

Reply:
Hi @dalessr , I have not done changes to any of the file except, UserMapData.json. Can you please what i should do before pushing the new changes.